src/style.css

```
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  color: #404e5f;
+  box-sizing: border-box;
+}
+
+input[type="search"]::-webkit-search-decoration,
+input[type="search"]::-webkit-search-cancel-button,
+input[type="search"]::-webkit-search-results-button,
+input[type="search"]::-webkit-search-results-decoration {
+  -webkit-appearance: none;
+
+}
+
+.wrapper {
+  margin: 20px 0;
+  position: relative;
+}
+
+.label {
+  position: absolute;
+  display: block;
+  width: 20px;
+  height: 20px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 20px;
+}
+
+.label--big {
+  top: 25px;
+  left: 26px;
+  background-size: 19px;
+}
+
+.label--small {
+  top: 15px;
+  left: 13px;
+  background-size: 11px;
+}
+
+.search {
+  font-family: "Avenir-Book", sans-serif;
```

You can use inherit value
```suggestion
  font-family: inherit;
```